ApollosProject/apollos-apps
# fix(community): update presentation to remove double header

## ✏️ Description
- fixes an issue where Find Community shows header twice
- updates presentation style from `push` to `card` as per migration guide
  - https://reactnavigation.org/docs/upgrading-from-5.x/#breaking-changes-from-react-native-screensnative-stack



## 🔬 To Test
1. Go to Community tab
2. Go to Find Community
3. Top header should not be doubled

## 📸 Screenshots
![image](https://user-images.githubusercontent.com/7557597/215569555-a7b14796-8320-4bce-a68f-cf4dc90bc142.png)



## ⚠️ To-do Before Merge



## 📝 Checklist
